crates/core/component/dex/src/circuit_breaker/value.rs

```diff
@@ -0,0 +1,69 @@
+use std::collections::BTreeMap;
+
+use penumbra_asset::{asset, Value};
+use penumbra_num::Amount;
+use penumbra_proto::{core::component::dex::v1alpha1 as pb, DomainType, TypeUrl};
+
+#[derive(Debug, Clone, Default)]
+pub struct AssetTallies {
```

Comment:
This isn't used AFAICT?

Reply:
This is preliminary for the value circuit breaker